src/Sentry.AspNetCore/Extensions/HttpContextExtensions.cs

```diff
@@ -43,6 +47,34 @@ internal static class HttpContextExtensions
             return null;
         }
 
+        // Internal for testing.
+        internal static string ReplaceMcvParameters(string route, HttpContext? context)
+        {
+            // Return RouteData or Null, marking the HttpContext as nullable since the output doesn't
+            // shows the nullable output.
+            var routeData = context?.GetRouteData();
+
+            if (routeData?.Values["controller"]?.ToString() is { } controller)
+            {
+                route = Regex.Replace(route, "\\{controller=[^\\}]+\\}", controller);
+            }
+
+            if (routeData?.Values["action"]?.ToString() is { } action)
+            {
+                route = Regex.Replace(route, "\\{action=[^\\}]+\\}", action);
+            }
+
+            if (routeData?.Values["area"]?.ToString() is { } area)
+            {
+                route = Regex.Replace(route, "\\{area=[^\\}]+\\}", area);
```

Comment:
Please remove the `\\`:

```suggestion
                route = Regex.Replace(route, "{area=[^}]+}", area);
```